dapr/clients/grpc/client.py

```diff
@@ -1088,6 +1093,263 @@ def unlock(
         return UnlockResponse(status=UnlockResponseStatus(response.status),
                               headers=call.initial_metadata())
 
+    def start_workflow(
```

Comment:
Aside the exception being caught in the catch blocks, are there major differences between the AIO and non-AIO versions? Could this be in a mixin class?

Reply:
NVM, we tag the async functions with `async`. A mixin would not work here :(